docs/integrations-guides/rollup-guides/op-stack/README.md

```diff
@@ -22,7 +22,10 @@ First check out the version of the EigenDA proxy corresponding to the version of
 |------------------|-----------------------|
 | [v1.7.6](https://github.com/ethereum-optimism/optimism/releases/tag/v1.7.6)           | [v1.0.0](https://github.com/Layr-Labs/eigenda-proxy/releases/tag/v1.0.0)                |
 | [v1.7.7](https://github.com/ethereum-optimism/optimism/releases/tag/v1.7.7)           | [v1.2.0](https://github.com/Layr-Labs/eigenda-proxy/releases/tag/v1.2.0)                |
-| [v1.9.0](https://github.com/ethereum-optimism/optimism/releases/tag/v1.9.0)           | [v1.4.0](https://github.com/Layr-Labs/eigenda-proxy/releases/tag/v1.4.0)
+| [v1.9.0](https://github.com/ethereum-optimism/optimism/releases/tag/v1.9.0)           | [v1.4.0](https://github.com/Layr-Labs/eigenda-proxy/releases/tag/v1.4.0)                |
+| [op-challenger v1.1.0](https://github.com/ethereum-optimism/optimism/releases/tag/op-challenger%2Fv1.1.0)           | TODO(@bxue): make new release                |
```

Comment:
TODO for release-captain @bxue-l2

Reply:
Thinking about this... I think eigenda-proxy's dependency on op-stack is very very light (we use it for metrics, their server, and e2e tests). Not sure we should show compatibility version here since afaiu it's not a hard dependency at all. Is "compatibility" even necessary? cc @epociask